openmc/filter.py

```diff
@@ -1324,6 +1324,19 @@ def check_bins(self, bins):
             cv.check_greater_than('filter value', v0, 0., equality=True)
             cv.check_greater_than('filter value', v1, 0., equality=True)
 
+    def bin_lethargy(self):
```

Comment:
Let's make this a property. Also, I'd suggest the following name:
```suggestion
    @property
    def lethargy_bin_width(self):
```

Reply:
You'll also need to add a description of this new property to the class docstring